src/Helpers/DocumentCreation/DocumentCreationHelper.cs

```diff
@@ -15,18 +16,32 @@ public List<string> GenerateQuestionAndAnswersList(FormAnswers formAnswers, Form
         {
             var summaryBuilder = new SummaryAnswerBuilder();
 
-            var formSchemaQuestions = formSchema.Pages.ToList()
+            var formSchemaQuestions = formSchema.Pages
                 .Where(_ => _.Elements != null)
                 .SelectMany(_ => _.ValidatableElements)
                 .ToList();
 
-            formSchemaQuestions.ForEach(question =>
+            formSchema.Pages.ForEach(page =>
             {
-                var answer = _elementMapper.GetAnswerStringValue(question, formAnswers);
+                formSchemaQuestions.ForEach(question =>
+                {
+                    var answer = _elementMapper.GetAnswerStringValue(question, formAnswers);
 
-                summaryBuilder.Add(question.GetLabelText(), answer, question.Type);
+                    if (question.Type.Equals(EElementType.Address)
+                        || question.Type.Equals(EElementType.Street)
+                        || question.Type.Equals(EElementType.Organisation))
```

Comment:
instead of doing this logic would it not be easier to just pass in the pagetitle and override in each elements GetLabelText to use the pageTitle instead, would remove the need for this if logic.

Reply:
I did think about that, I was 50/50 about which way to go